### DISCUSSION TITLE
Different Levels

### DISCUSSION THREAD

**Discussion by Overseer24** [cjmellor/level-up#66] [Q&A]
We are building a thesis final project similar to Reddit but for student hobbies and interests. This package will be a great help for me but I have a question, can we add different levels for each user in different communities? Like the user having different levels and level gaining on each community he/she joined into and make interactions with? Thank you

**Comment by cjmellor** [comment]
Hi,

Out of the box, no, you cannot do this with the package, but it should be do-able, I would suggest you fork the repo and make adjustments to get it how you want it.
